src/api/requestBuilders/DeleteRequestBuilder.ts

```diff
@@ -1,7 +1,6 @@
 import { BaseRequestBuilder } from './BaseRequestBuilder';
 
+/** @public */
 export class DeleteRequestBuilder extends BaseRequestBuilder {
-  constructor(url: string) {
-    super('DELETE', url);
-  }
+  protected override method = 'DELETE';
```

Comment:
Just wondering, why use this `override` vs the `super` call? Coming from Java, this would be done in the call to `super`, so I'm curious about javascript class conventions.